### ISSUE TITLE
Training failure

### ISSUE THREAD

**Issue by SunInHeart** [zhshi0816/Video-Frame-Interpolation-Transformer#7]
Thanks for your excellent work.
When i trained this model, the loss quickly increased and got nan. I just resized the batch_size to 2.
![7cf46c7179d2612e7e8560f89b74ada6](https://user-images.githubusercontent.com/63057663/196023180-dabfee02-0621-4bec-87c5-491b54aac70a.png)
And the loss grows slowly after changing the learning rate to 1e-5.
![8be608d7891c4809dd05639db57c3def](https://user-images.githubusercontent.com/63057663/196023355-295023a2-7ad5-42d0-b16f-f4479f6de393.png)
I wonder why this is happening.

**Comment by zhshi0816** [comment]
Hi,

Thanks for following. I didn't observe this problem on my side. What's your environment? Could you align your environments to ours?

**Comment by SunInHeart** [comment]
Hi,
Thanks for your reply. I tried it on the Linux server and found no problem. But on the Windows OS, the question is still unsolved. 
The environment  is the same for both.

**Comment by zhshi0816** [comment]
I did all the experiments on Linux, never try on the Windows. Maybe there are some differences between them.
